src/services/infinispanRefData.ts

```diff
@@ -210,3 +210,9 @@ export enum ConfigDownloadType {
   XML = 'XML',
   YAML = 'YAML',
 }
+
+export enum DataDistributionStatsOption {
+  Entries = 'Total number of entries',
```

Comment:
This should be in the translation so the enum can be used that way

Reply:
Using translation there is throwing an error:
`Uncaught Error: Invalid hook call. Hooks can only be called inside of the body of a function component`

and also enum does not allow computed values.

Reply:
you can't use translation there, you need to be in the react component. you could use a label name instead